### PR TITLE
center modals when a quick start is open

### PR DESCRIPTION
**Pull request by jschuler**

### CODE REVIEW

**Comment by pmuir** [comment]
@srambach are you able to review this one?

File: src/app/App.scss [hunk 1]
```diff
@@ -205,4 +205,9 @@
   border-right: 1px solid var(--pf-c-select__toggle--before--BorderRightColor);
   border-bottom: 1px solid var(--pf-c-select__toggle--before--BorderBottomColor);
   border-left: 1px solid var(--pf-c-select__toggle--before--BorderLeftColor);
-}
\ No newline at end of file
+}
+
+// centers modals when a quick start drawer is open
+.pf-c-drawer.pf-m-expanded .pfext-quick-start-drawer__body .pf-c-backdrop {
+  position: absolute;
```
Comment (srambach) [inline src/app/App.scss:212]: Looks like there is a variable here we can use `--pf-c-backdrop--Position: absolute;`
and, possibly the selector can just be `.pfext-quick-start-drawer__body .pf-c-backdrop`? 

Inside kafka, I see the modals overlay the entire screen including quickstart. Is that preferable and possible? 
image

Comment (jschuler) [reply src/app/App.scss:212]: The goal is to keep the quick start drawer on the right in view and not blocked out by the modal. That is so a user can follow instructions that might take them through modals. For that to happen, 2 things need to be done:
1. modals append themselves to the `#chrome-app-render-root`
2. the backdrop needs to be absolute and not fixed

I've made the first change to some of the modals, like the 'Create kafka instance' modal.
Screen Shot 2021-11-04 at 1 50 07 PM

Changing the backdrop position to absolute will center the modal:
Screen Shot 2021-11-04 at 1 50 49 PM

I can see if the `--pf-c-backdrop--Position` variable can be changed to do that instead though!

Comment (srambach) [reply src/app/App.scss:212]: Sounds good :) And try reducing the selector; that was a very high specificity and I think you can get away with just scoping it inside the quickstart drawer body but I'm not positive.

File: package.json [hunk 2]
```diff
@@ -36,9 +36,9 @@
     "clean": "rimraf dist"
   },
   "dependencies": {
-    "@patternfly/patternfly": "4.132.2",
-    "@patternfly/react-core": "4.152.4",
-    "@patternfly/react-table": "4.29.58",
+    "@patternfly/patternfly": "4.151.4",
```
Comment (jschuler) [inline package.json:39]: update so i can use the newer `--pf-c-backdrop--Position` variable, and also match insights-chrome
https://github.com/RedHatInsights/insights-chrome/blob/65b1408ebbcc29edc548e0fe7043542889fe1040/package.json#L136

**Comment by srambach** [comment]
Looks great! Thanks! 👍

**Pull request merged**